.env.ci

```diff
@@ -10,24 +10,29 @@ DB_NAME=$DB_NAME
 
 TYPEORM_LOGGING=error,migration,warn,info
 
+# REDIS
+REDIS_VERSION=6.0.8
+REDIS_PASSWORD=$REDIS_PASSWORD
+REDIS_PORT=$REDIS_PORT
+REDIS_HOST=$REDIS_HOST
```

Comment:
Shouldn't Redis be launched on localhost, when devloping locally?

Reply:
it is: https://github.com/energywebfoundation/ssi-hub/blob/543ed0cfffdfd882ca6a790996b6c3e3b1748a64/docker-compose.dev.yml#L28-L28

Reply:
`.env.ci` and `compose.dev.yml` references `REDIS_*` variables, but where these variables are actually set?

Reply:
I believe that `$REDIS_HOST` and `$REDIS_PORT` are set in GH action. I just moved this part of this file to make its structure the same as .env.dev. It is easier to compare them now.

.env.dev has just values here: https://github.com/energywebfoundation/ssi-hub/blob/543ed0cfffdfd882ca6a790996b6c3e3b1748a64/.env.dev#L12-L12

Reply:
we don't use .env in GH now. These files serves only as config for local develop/testing

Reply:
So, probably `.env.ci` is not necessary at all and could be removed to not distract people. What do you think, @JGiter

Reply:
Yes, we don't have `ci` environment. We would better replace .env.ci with .env.staging, .env.dist with .env.prod and create .env.sandbox to match environments

Reply:
or, maybe we should just leave one .env.example as a template and remove all others?

Reply:
it will be invonvenient. For example if you want to debug `prod`, then you need to pick `prod` values from `.env.example` and put them in `.env`. Simplre just copy `.env.prod` to `.env`

Reply:
I see. I was thinking more like preparing env vars for deployments, not connecting to production or staging infrastructure.